README.md

```diff
@@ -1,7 +1,50 @@
 # Frost Trusted Dealer Demo
 
-A CLI demo for running trusted dealer key generation with FROST
+A CLI demo for running trusted dealer key generation with the Zcash Foundation's Rust implementation of ['Two-Round Threshold Schnorr Signatures with FROST'](https://datatracker.ietf.org/doc/draft-irtf-cfrg-frost/).
 
-This demo uses ed25519
+This will be part of a set of demos and a proof of concept application that uses the FROST libraries and reference implementation. The purpose of this is:
 
-https://crates.io/crates/frost-ed25519
+1. To identify gaps in our documentation and provide usage examples for developer facing documentation.
+2. To provide reference implementations for developers wanting to use FROST in a “real world” scenario.
+
+This demo uses the (Ed25519, SHA-512) ciphersuite. The crate can be found here: https://crates.io/crates/frost-ed25519
+
+## Status ⚠
+
+The Trusted Dealer demo is a WIP.
+
+## Usage
+
+NOTE: This is for demo purposes only and should not be used in production.
+
+On startup, the Trusted Dealer demo will prompt for:
+
+1. Minimum number of signers (>= 2) i.e. The threshold number of participants for the secret sharing scheme
+2. Maximum number of signers i.e. the number of shares to generate
+
+The dealer CLI will then use that data to generate:
+
+1. The group public key
+2. A commitment to the secret
+3. Each participant’s public key
+
+This above data will be output by the Trusted Dealer demo to the terminal.
```

Comment:
Maybe we want to make some comment about how in a "real world" scenario, the secret share should be delivered securely to each participant and not simply printed out to terminal. Additionally, or instead, we might want to explain what each of these items which are output is used for in the next round; which should be kept secret and which should be kept private and how they would ideally be delivered to participants.

I know this is all in the spec but people are lazy about reading links :D

Reply:
Done